store/src/java/com/zimbra/cs/service/account/ChangePassword.java

```diff
@@ -112,6 +139,6 @@ public boolean needsAuth(Map<String, Object> context) {
         // them the means to do that. We cannot rely on the Provisioning
         // changePassword check alone, or we bypass 2FA protection.
 
-        return true;
+        return false;
```

Comment:
I'm not sure we need to change this. Wouldn't it just boot them out earlier if they sent no auth token? In any case, you should probably change/delete the comments here to reflect the new behavior.